Lib/test/test_format.py

```diff
@@ -495,6 +495,10 @@ def test_g_format_has_no_trailing_zeros(self):
         self.assertEqual(format(12300050.0, ".6g"), "1.23e+07")
         self.assertEqual(format(12300050.0, "#.6g"), "1.23000e+07")
 
+    def test_with_two_commas_in_format_specifier(self):
+        error_msg = "Cannot specify ',' with ','."
```

Comment:
Technically, because this is a regex, the '.' should be escaped. Although calling re.escape() is probably easiest. It's always bugged me that there's not a version of assertRaisesRegex that doesn't require a regex.

Reply:
@ericvsmith  For testing the original error message, is it Okay to do something along the lines of

```
 @unittest.expectedFailure
    def test_with_two_commas_in_format_specifier_for_old_error_msg(self):
        old_error_msg = re.escape("Cannot specify both ',' and '_'.")
        with self.assertRaisesRegex(ValueError, old_error_msg):
```
?

Reply:
Or
```
...
old_error_msg = "Cannot specify both ',' and '_'."
with self.assertRaises(ValueError) as cm:
    f'{1:,,}'
self.assertNotEqual(str(cm.exception), old_error_msg)
...
```

Reply:
I meant testing that something like `f'{1:,_}'` actually raises the exception, and still does so after your change. Just for grins, you might want to test it both ways, so also `f'{1:_,}'`.

Reply:
Ah, that. Sure. Sorry, i was confused :)